packages/node/src/executors/execute/execute.impl.spec.ts

```diff
@@ -34,13 +35,16 @@ describe('NodeExecuteBuilder', () => {
     );
 
     fork.mockImplementation(() => {
-      return {
-        on: (eventName, cb) => {
+      mockSubProcess = {
+        on: jest.fn().mockImplementation((eventName, cb) => {
```

Comment:
Represents a breaking change: the type of event yielded by executeExecutor is now a union of of types, see: https://github.com/nrwl/nx/pull/7110/files#diff-8b1daad050dbc745646b89f91cd20c497a23f51caf7eeaddb96a517a80b8f01fR28